Config/Engine.ini

```diff
@@ -1,3 +1,10 @@
+# UE 5.3+ defaults this to 11, which is not really high enough.
+# 100 is probably high enough for reasonable values of Cesium3DTileset's 'MaximumSimultaneousTileLoads property,
+# but if you get log messages like "Warning: Reached threaded request limit", and you're sure you want to use
+# such a high value, you may need to increase this.
+[HTTP.HttpThread]
+RunningThreadedRequestLimit=100
+
```

Comment:
100 simultaneous HTTP requests sounds really high to me. Is this something that users really need?

There are some discussions about web browser parallel connection [here](https://stackoverflow.com/questions/985431/max-parallel-http-connections-in-a-browser), and typical values are less than 10.

In my personal experience with writing client code that does parallel fetches, I typically settled with 4. Values that were higher, like 8, 12, 16 showed diminishing returns with improving load time.

Reply:
The number of simultaneous requests is a setting in the UI (maximum simultaneous tile loads), and it defaults to 20. More simultaneous tile loads makes a _huge_ difference in some cases. For example, from here in Australia we have good bandwidth but very high latency, so downloading lots of tiles in parallel is a big win. In any case, this value in the config file needs to be higher than the value a user could _reasonably_ try to use, or else they'll get a warning. It doesn't imply we'll actually _do_ 100 simultaneous requests.